packages/react-scripts/config/webpackDevServer.config.js

```diff
@@ -10,9 +10,7 @@
 
 const errorOverlayMiddleware = require('react-dev-utils/errorOverlayMiddleware');
 const evalSourceMapMiddleware = require('react-dev-utils/evalSourceMapMiddleware');
-const noopServiceWorkerMiddleware = require('react-dev-utils/noopServiceWorkerMiddleware');
 const ignoredFiles = require('react-dev-utils/ignoredFiles');
-const redirectServedPath = require('react-dev-utils/redirectServedPathMiddleware');
```

Comment:
These two files were added to `react-dev-utils`, a sibling to `react-scripts`. I don't understand their build process, though, since the version number of `react-dev-utils` was not incremented.

One option would be to publish our own fork of `react-dev-utils` alongside this `react-scripts` fork, but since these middlewares are very small, I opted to just inline the contents into this file to remove the dependency.